jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/rds/RdsExplorerNodes.kt

```diff
@@ -5,28 +5,33 @@ package software.aws.toolkits.jetbrains.services.rds
 
 import com.intellij.openapi.project.Project
 import software.amazon.awssdk.services.rds.RdsClient
-import software.amazon.awssdk.services.rds.model.DBInstance
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerNode
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerResourceNode
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerServiceNode
 import software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerServiceRootNode
 import software.aws.toolkits.jetbrains.core.getResourceNow
+import software.aws.toolkits.jetbrains.core.utils.buildMap
+import software.aws.toolkits.jetbrains.services.rds.resources.LIST_SUPPORTED_CLUSTERS
 import software.aws.toolkits.jetbrains.services.rds.resources.LIST_SUPPORTED_INSTANCES
 
 class RdsExplorerParentNode(project: Project, service: AwsExplorerServiceNode) : AwsExplorerServiceRootNode(project, service) {
-    override fun getChildrenInternal(): List<AwsExplorerNode<*>> = nodeProject.getResourceNow(LIST_SUPPORTED_INSTANCES).map {
+    override fun getChildrenInternal(): List<AwsExplorerNode<*>> = buildMap<String, RdsDatabase> {
+        // De-dupe by db identifier
```

Comment:
Add a comment that clusters take priority? Or does it matter? Is this the order the console does?

Reply:
Console builds a tree, and puts instances under the cluster. (Show it shows everything)